style.css

```diff
@@ -106,12 +107,13 @@ img {
 
 /* Text */
 
-h1 {
+.main-headings {
```

Comment:
H1?

Reply:
for accessibility h1 is used in the header for the logo text and then used again in the hero, so changing this one breaks both.

Reply:
changed logo text to span and implemented h1 instead of classes.